examples/with-clerk/README.md

```diff
@@ -0,0 +1,15 @@
+
+# Clerk.dev Example
+
+Add sign up, sign in, and profile management to your Next.jsapplication in minutes with [Clerk.dev](https://www.clerk.dev/?utm_source=github&utm_medium=nextjs-examples&utm_campaign=nextjs).
+
+## Getting Started
+
+Learn to install and initialize Clerk in a new Next.js application by reading our [official guide](https://docs.clerk.dev/frontend/quick-starts/next.js).
+
+## Learn More
```

Comment:
Please follow the format of the other examples in the README 😄

Reply:
Done.

Reply:
Oops my bad. Done.